packages/protocol/contracts/L1/tiers/TierProviderBase.sol

```diff
@@ -57,6 +57,17 @@ abstract contract TierProviderBase is ITierProvider {
             });
         }
 
+        if (_tierId == LibTiers.TIER_ZKVM_RISC0) {
+            return ITierProvider.Tier({
+                verifierName: LibStrings.B_TIER_ZKVM_RISC0,
+                validityBond: 250 ether, // TKO
```

Comment:
Lets change the validityBond to a value between 125 and 250, for example, 200 TKO.

Reply:
Do the other parameters take the same middle values, Like contestBond?